src/SpillRegisters_X64.S

```diff
@@ -0,0 +1,27 @@
+.text
+
+.extern scan_stack
+.globl spill_registers
+
+spill_registers:
+    // Push all callee-saves registers to get them on the stack for conservative
+    // stack scanning. We maintain 16-byte alignment at calls (Sys V ABI). There
+    // is an 8-byte return address on the stack and we push 56 bytes which
+    // maintains 16-byte stack alignment at the call.
+    push $0
```

Comment:
What's `$0` again?

Reply:
GAS syntax requires all constants to be prefixed with `$`. We're simply pushing padding to the stack to maintain 16-byte alignment.

[As an aside, I ran into a nasty gotcha with this one. I'd written `push 0` before, which actually pushes the contents of memory address 0, which, of course, segfaults as it tries to dereference it. :man_facepalming:  ]

Reply:
Ah yes. Good old GNU syntax.